src/improvements/tasks/IsNestedTask.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import {MultisigTask} from "src/improvements/tasks/MultisigTask.sol";
+import {AddressRegistry} from "src/improvements/AddressRegistry.sol";
+import {Script} from "forge-std/Script.sol";
+
+/// @title IsNestedTask
+/// @notice Template contract for checking if a task is nested
+contract IsNestedTask is Script {
+    /// @notice Returns Useful function to tell if a task is nested or not based on the task config
+    function isNestedTask(string memory taskConfigFilePath) public returns (bool) {
```

Comment:
Should this method just live in the base `MultisigTask.sol`, then we can just have this public method use the same logic used by `_setIsNestedSafe`?